NOTES.md

```diff
@@ -1,3 +1,76 @@
+# Release 2.0.0 (Tesuji)
+
+The highlights of this release are:
+* Support for attaching a changefeed to the `get_all` and `union` commands
+* Improved support for asynchronous queries
+
+Read the [release blog post][2.0-blog] for more details.
+
+[2.0-blog]: http://rethinkdb.com/blog/2.0-release/
+
+## Compatibility ##
+
+Data files from RethinkDB versions 1.13.0 onward will be automatically
+migrated to version 2.0. As with any major release, back up your data files
+before performing the upgrade.
+
+Additional compatibility notes and breaking changes are listed in the
+2.0.0-RC1 release notes.
+
+## Changes since 2.0.0-RC1
+
+### New features ###
+
+* Added an asynchronous API to the Python and Ruby drivers (#2622)
+* Reached a production-ready state (#1174, #1000)
+
+### Improvements ###
+
+* Simplified the `multi_throttling` infrastructure (#4021)
+* The server now reports incantation errors to client drivers earlier (#4011)
+* Set `TCP_NODELAY` in all drivers (#3998)
+
+### Bug fixes ###
+
+* Ruby driver: now handle signals correctly (#4029)
+* JavaScript driver: fixed a bug that could be triggered by casting `close` twice (#4017)
+* Fetching Browserify during the build process is now more reliable (#4009)
+* Fixed a bug in `rethinkdb export` to no longer hang when certain errors occur (#4005)
+* Fallback to TCP4 when binding sockets (#4000)
+* `order_by` changes are now returned in order (#3993)
```

Comment:
This should say:
Initial values for an `order_by` changefeed are now returned in order